docs/lava-blockchain/join-mainnet-manual-cosmovisor.md

```diff
@@ -0,0 +1,218 @@
+---
+sidebar_position: 2
+slug: /mainnet-manual-cosmovisor
+title: Option A - With Cosmovisor
+---
+import RoadmapItem from '@site/src/components/RoadmapItem';
+
+# Join mainnet - Manual setup with Cosmovisor
+## Prerequisites
+
+1. Verify [hardware requirements](reqs) are met
+2. Install package dependencies
+    - Note: You may need to run as `sudo`
+    - Required packages installation
+        
+        ```bash
+        ### Packages installations
+        sudo apt update # in case of permissions error, try running with sudo
+        sudo apt install -y unzip logrotate git jq sed wget curl coreutils systemd
+        # Create the temp dir for the installation
+        temp_folder=$(mktemp -d) && cd $temp_folder
+        ```
+        
+    - Go installation
+        
+        ```bash
+        ### Configurations
+        go_package_url="https://go.dev/dl/go1.20.5.linux-amd64.tar.gz"
+        go_package_file_name=${go_package_url##*\/}
+        # Download GO
+        wget -q $go_package_url
+        # Unpack the GO installation file
+        sudo tar -C /usr/local -xzf $go_package_file_name
+        # Environment adjustments
+        echo "export PATH=\$PATH:/usr/local/go/bin" >>~/.profile
+        echo "export PATH=\$PATH:\$(go env GOPATH)/bin" >>~/.profile
+        source ~/.profile
+        ```
+        
+    - Installation verifications
+        
+        
+        1. You can verify the installed go version by running: `go version`
+        
+        2. The command `go env GOPATH` should include `$HOME/go`
+        If not, then, `export GOPATH=$HOME/go`
+        
+        3. PATH should include `$HOME/go/bin`
+        To verify PATH, run `echo $PATH`
+        
+
+## 1. Set up a local node
+
+### Download app configurations
+
+- Download setup configuration
+    
+    Download the configuration files needed for the installation
+    
+    ```bash
+    # Download the installation setup configuration
+    git clone https://github.com/lavanet/lava-config.git
+    cd lava-config/mainnet
+    # Read the configuration from the file
+    # Note: you can take a look at the config file and verify configurations
+    source setup_config/setup_config.sh
+    ```
+    
+- Set app configurations
+        
+    Copy lavad default config files to config Lava config folder
+    
+    ```bash
+    echo "Lava config file path: $lava_config_folder"
```

Comment:
are we missing calling lavad init?

Reply:
Fixed

Reply:
We need to set the value for the moniker parameter in config.toml

Reply:
This is done in step 2